data/mods/DinoMod/DESIGN.md

```diff
@@ -0,0 +1,15 @@
+# Design Document SPOILERS AHEAD
+The core idea of DinoMod is this: https://tvtropes.org/pmwiki/pmwiki.php/Main/EverythingsBetterWithDinosaurs
+The purpose of this mod is to make Cataclysm a more fun game to play and develop for, showing off new mechanics and bringing life to parts of the game that aren't as well-developed.
+
+# What belongs in DinoMod?
+Dinosaurs and content based around dinosaurs that for whatever reason can't just go into the vanilla game.
+
+# Realism vs. Fun
+There is more space in mods for fun. In general the vanilla living dinos should be as realistic as possible to the science we know but the zombie, CBM, and fungal dinos can go anywhere as long as it's fun.
+
+# Adding, replacing, removing
+This mod is about adding content. In general it should have as light a touch on the vanilla game as possible. This helps us keep it working by itself and with other mods.
+
+# How to contribute
```

Comment:
This section probably belongs in the DinoMod README. Design documents are usually to outline the overall theme and goals of a project. 🆗